### PR TITLE
EIP-4938: Move to back to draft

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**
This PR was falsely moved to stagnant, move to last call instead

### CODE REVIEW

File: EIPS/eip-4938.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "eth/67: Removal of GetNodeData"
 description: "Remove GetNodeData and NodeData messages from the wire protocol"
 author: Marius van der Wijden (@MariusVanDerWijden), Felix Lange <fjl@ethereum.org>, Gary Rong <garyrong@ethereum.org>
 discussions-to: https://ethereum-magicians.org/t/eip-4938-removal-of-getnodedata/8893
-status: Stagnant
+status: Last Call
```
Comment (xinbenlv) [inline EIPS/eip-4938.md:7]:
```suggestion
status: Draft
```
Hi @MariusVanDerWijden thank you for authoring and continue to drive this EIP. Happy to see this EIP being revived. While looking at the history, the EIP was Draft moved to Stagnant, hence update the status to Draft to revive. Once it's revived to Draft, feel free to create PR to move to status: Review -> Last Call -> Final.

I know this process probably is a bit longer. Please let us know if this EIP is already widely adopted / implemented by client and I could try to help bring to the EIP Editor's meeting to advocate for an expedite procedure.

Comment (xinbenlv) [reply EIPS/eip-4938.md:7]: Please also take a look at the linter error and try to fix all the linter errors

**Review by Cryptonics1 — APPROVED** [review]

File: EIPS/eip-4938.md (hunk 1 above)
Comment (MariusVanDerWijden) [reply EIPS/eip-4938.md:7]: It's been implemented by all execution layer clients

Comment (lightclient) [reply EIPS/eip-4938.md:7]: It does need to go back to draft unfortunately, sorry :(

Comment (MariusVanDerWijden) [reply EIPS/eip-4938.md]: moved to draft

**Review by lightclient — APPROVED** [review]

**Comment by lightclient** [comment]
Waiting on #6034

**Comment by axic** [comment]
> Waiting on https://github.com/ethereum/EIPs/pull/6034

This was merged.

**Comment by SamWilsn** [comment]
You'll need to replace `master` in your devp2p links with a specific commit.

---

Until #6091 is merged, `eipw` will keep complaining.

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**